governance/internationals.md

```diff
@@ -0,0 +1,55 @@
+# International Bible Quizzing
+These rules apply to the Internationals quiz meet conducted once yearly, administered under the oversight of the CQLT.
+
+## Eligibility Requirements
+- All quizzers for International competition must be 12 to 18 years of age by the meet registration date or have been enrolled as a full-time high school student during the current quiz year.
+- In order to quiz at Internationals for a specific district, the quizzer must reside in that district or in a bordering community.  Their home church must be from the district in which they quiz.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+
+## Team Structure
+- Each district will be allowed to send as many teams as the district feels will be competitive and that they are financially able to send.  These will be the district International Teams.
+- A team will be composed of a maximum of five members: four starting quizzers and one substitute.  These five shall constitute the team for International Finals.  Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).
+- Each team must have a coach and may also have an assistant coach. 
+
+## Event Registration
+- All arrangements, promotion, and administration will be by the Life Impact Office.
+- Notification by each district of its intention to enter a team must be received by the Life Impact Office, on a form supplied by that office, no later than the date posted on the C&MA Quizzing website.
```

Comment:
LIM or the CQLT?

Reply:
This occurs simultaneously. I would change to CQLT.